geometry/optimization/graph_of_convex_sets.cc

```diff
@@ -167,6 +177,46 @@ std::unordered_set<Edge*> GraphOfConvexSets::Edges() {
   return edges;
 }
 
+// TODO(russt): We could get fancy and dim the color of the nodes/edges
+// according to phi, using e.g. https://graphviz.org/docs/attr-types/color/ .
+std::string GraphOfConvexSets::GetGraphvizString(
+    const std::optional<solvers::MathematicalProgramResult>& result,
+    bool show_slacks, int precision, bool scientific) const {
+  // Note: We use stringstream instead of fmt in order to control the
+  // formatting of the Eigen output and double output in a consistent way.
+  std::stringstream graphviz;
+  graphviz.precision(precision);
+  if (!scientific) graphviz << std::fixed;
+  graphviz << "digraph GraphOfConvexSets {\n";
+  graphviz << "labelloc=t;\n";
+  for (const auto& vpair : vertices_) {
+    graphviz << "v" << vpair.first << " [label=\"" << vpair.second->name();
+    if (result) {
+      graphviz << "\n x = "
+               << result->GetSolution(vpair.second->x()).transpose();
```

Comment:
Minor: here it'd be nice to show `x` with parentheses and commas, e.g. `x = (1.000, 2.000)` instead of  `x = 1.000 2.000`. It took me a couple of second to realize they were the entries of `x`.